capa/features/extractors/ida/file.py

```diff
@@ -143,6 +143,16 @@ def extract_file_strings():
             yield String(s.s), (seg.start_ea + s.offset)
 
 
+def extract_file_library_functions():
+    """
+    extract the names of statically-linked library functions.
+    """
+    if False:
```

Comment:
```suggestion
    for ea in idautils.Functions():
	    if idaapi.get_func(ea).flags & idaapi.FUNC_LIB:
		    name = idaapi.get_name(ea)
		    yield capa.features.insn.API(name), ea
```

Reply:
I think the formatting is off with my suggestion but we could use this code to identify library function names as see by IDA.